src/main/webapp/resources/js/pages/clients/ClientsPage.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
```

Comment:
File comments

Reply:
Updated in 09b609af6270c3c4ebbe2c0b083bf510581fa314